### ISSUE TITLE
League managers under “about fcc -> contact”

### ISSUE THREAD

**Issue by amiller47** [Hazmit/fcc_register_bugs#52]
Seems hidden away.....maybe move to league pages

**Comment by dhejna** [comment]
Could League Managers be another 'flag' that we have on our user profiles? (we may have chatted about this already)

**Comment by Hazmit** [comment]
Yes I think if you check in github there might be a feature request for
this already. If there is not would you make one for me?

I agree we should track league managers this way.

**Comment by Hazmit** [comment]
Resolved with new option in league to search and set a user as a league manager.
